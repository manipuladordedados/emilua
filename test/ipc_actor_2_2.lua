-- serialization/bad
local badinjector = require 'ipc_actor_badinjector'

local function gen_oversized_table()
    local ret = {}
    for i = 1, badinjector.CONFIG_MESSAGE_MAX_MEMBERS_NUMBER + 1 do
        ret[i .. ''] = '.'
    end
    return ret
end

local my_channel = spawn_vm('', { subprocess = {} })
my_channel:send(gen_oversized_table())
