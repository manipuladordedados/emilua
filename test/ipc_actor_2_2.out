Main fiber from VM 0x1 panicked: 'EINVAL'
stack traceback:
	[string "?"]: in function 'send'
	input.lua:13: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
